projects/experimental/directives/pin/pin-group.directive.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    AfterViewInit,
+    ChangeDetectorRef,
+    ContentChildren,
+    Directive,
+    Inject,
+    QueryList,
+    Self,
+} from '@angular/core';
+import {TuiDestroyService} from '@taiga-ui/cdk';
+import {Observable, Subject} from 'rxjs';
+import {filter, takeUntil} from 'rxjs/operators';
+
+import {TuiPinDirective} from './pin.directive';
+
+@Directive({
+    selector: '[tuiPinGroup]',
```

Comment:
Let's drop this directive for now and openChange so that for the first iteration pins are only manually open-able.

Reply:
fixed